### PR TITLE
Database

### PR DESCRIPTION
**Pull request by chiijlaw**
@lancl Please review my PR whenever you have free time. Thanks so much!

### CODE REVIEW

File: database/database.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const mongoose = require('mongoose');
 
 mongoose.connect(
-  'mongodb://localhost/vourcher',
+  'mongodb://172.17.0.2/vourcher',
```
Comment (lancl) [inline database/database.js:4]: Is '172.17.0.2' the hostname that you chose, for storing your DB on Amazon cloud?

Comment (chiijlaw) [reply database/database.js:4]: 172.17.0.2 corresponds to the IP within the EC2 instance that let's my microservice's container connect to the mongodb database that is also within that same EC2 instance.

File: .elasticbeanstalk/config.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+branch-defaults:
```
Comment (lancl) [inline .elasticbeanstalk/config.yml:1]: Did you write this yml file? or was this generated by the system?

**Review by lancl — APPROVED** [review]
All looks good! Left some comments/questions.

File: .elasticbeanstalk/config.yml (hunk 2 above)
Comment (chiijlaw) [reply .elasticbeanstalk/config.yml:1]: This was generated by awsebcli when I ran the command eb init.

**Pull request merged**